modules/auxiliary/gather/crushftp_authbypass_cve_2025_2825.rb

```diff
@@ -0,0 +1,119 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'CrushFTP AWS4-HMAC Authentication Bypass',
+        'Description' => %q{
+          This module leverages an authentication bypass in CrushFTP 11 < 11.3.1 and 10 < 10.8.4. Attackers
+          with knowledge of a valid username can provide a crafted S3 authentication header to the CrushFTP web API
+          to authenticate as that user without valid credentials. When successfully executed, the exploit will
+          output working session cookies for the target user account. This vulnerability is tracked as CVE-2025-2825.
```

Comment:
```suggestion
          output working session cookies for the target user account.
```

This information is already in `References -> CVE`